docs/whatsnew_1.5.7.html

```diff
@@ -173,19 +173,34 @@ <h2>
+                <li>Core - Updated hundreds of files for compatibility with PHP 7.2, 7.3 and 7.4.  <b>These changes, along with updating your PHP version, will make your cart faster.</b></li>
                 <li>Admin - Default notification behavior for order update (send email/do not send email/hide update) may now be set</li>
                 <li>Admin - Admins can now login to customer accounts to place orders</li>
                 <li>Admin - Added product-details popover to allow quick visibility into products bought in an order without opening the order</li>
                 <li>Admin - Validation for integer min and max values and email addresses</li>
                 <li>Admin - Add sort order to order statuses so the admin order update dropdown can mirror your business workflow</li>
                 <li>Admin - Improve many configuration setting descriptions</li>
-                <li>Admin - Help buttons now link to new documentation</li>
+                <li>Admin - On-page help added, with links to new documentation</li>
+                <li>Admin – Products to Categories Link Manager. Addition of global tool to copy linked categories to another product.</li>
+                <li>Admin – Product Copy. Expanded to include copy of metatags and linked categories.</li>
+                <li>Admin - removed "Dear" from email templates to facilitate localization.</li>
+                <li>Admin - Improved customer search.</li>
+                <li>Admin - Improved error messaging.</li>
+                <li>Admin - Alphabetical sorting of Reports and Tools menus.</li>
+                <li>Admin - Improved sanitization of data.</li>
+                <li>Admin - Validation of configuration settings.</li>
+                <li>Admin - Improvements in Option Name/Value management.</li>
                 <li>Storefront - products with single-valued attributes (such as downloads) can now be added to cart from the listing page</li>
                 <li>Storefront - Product page now includes Ask a Question</li>
                 <li>Storefront - Contact Us page now includes a telephone number</li>
-                <li>Email - SPAM prevention built in to forms</li>
+                <li>Storefront - Improved handling of product restrictions such as units and minimum quantities</li> 
+                <li>Storefront - Define page added to shopping cart page for static content</li> 
+                <li>Email - SPAM prevention built in to forms (Credits @davewest for zenNonCAPTCHA)</li>
+                <li>Email - Improvements made to increase email deliverability</li>
+                <li>Email – hard-coded text removed.
                 <li>Checkout - Updated Square Payment module.  Start taking credit card payments in just 5 minutes! See www.zen-cart.com/partners/square_setup </li>
+                <li>Checkout - layout issues in responsive_classic template fixed.</li>
```

Comment:
These were mobile-only layout changes, specific to order-totals. Let's be clear about that.

Reply:
Changed, but this is not entirely true. #3094 also corrects the shipping page in mobile.  And  #2856 fixes the payment page on the desktop